### PR TITLE
Add new jobs

### PR DESCRIPTION
**Pull request by rbillings**
Addresses issue #67 , test for adding new jobs.

### CODE REVIEW

File: tests/test_add_new_jobs.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    page.result_sets[0].add_new_jobs()
+    assert len(page.result_sets[0].runnable_jobs) > 0
+
+@pytest.mark.nondestructive
+def test_hide_runnable_jobs(base_url, selenium):
+    page = TreeherderPage(base_url, selenium).open()
+    assert len(page.result_sets) > 1
+    page.result_sets[0].view()
+    page.result_sets[0].add_new_jobs()
+    assert 0 < len(page.result_sets[0].runnable_jobs)
```
Comment (davehunt) [inline tests/test_add_new_jobs.py]: Nit: We should keep this consistent with the assert from `test_add_new_jobs`, and assert `> 0`

File: tests/test_add_new_jobs.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import pytest
+
+from pages.treeherder import TreeherderPage
+
+
+@pytest.mark.nondestructive
+def test_add_new_jobs(base_url, selenium):
+    page = TreeherderPage(base_url, selenium).open()
+    assert len(page.result_sets) > 1
```
Comment (davehunt) [inline tests/test_add_new_jobs.py]: Nit: Both here and on line 21, I think we can assert that there's at least one result set, as we only use the first one. You could make this `> 0` or `>= 1`.

File: tests/test_add_new_jobs.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    page.result_sets[0].view()
+    page.result_sets[0].add_new_jobs()
+    assert len(page.result_sets[0].runnable_jobs) > 0
+
```
Comment (davehunt) [inline tests/test_add_new_jobs.py]: We need an additional blank line between the two tests.

**Comment by davehunt** [comment]
Looks good, just a couple of nits that aren't blockers. Also there's a flake8 issue. Consider this r+ with at least the flake8 issue addressed. Note that we should squash the commits on merge if someone else takes care of this.

**Comment by rbillings** [comment]
Squashed after fixing flake8 issues.

**Pull request merged**